XenAdmin/Wizards/PatchingWizard/PlanActions/ApplyXenServerPatchPlanAction.cs

```diff
@@ -67,7 +68,8 @@ protected override void RunWithSession(ref Session session)
             }
             else
             {
-                log.ErrorFormat("Mapping not found for patch {0} on master {1}", xenServerPatch.Uuid, master.uuid);
+                if (xenServerPatch != null && master != null)
+                    log.ErrorFormat("Mapping not found for patch {0} on master {1}", xenServerPatch.Uuid, master.uuid);

```

Comment:
I didn't deem it necessary because we're throwing the exception and the error will be recorded anyway, so I thought that an extra log line without any useful information such as the patch id wouldn't offer much.